shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-parser/src/main/antlr4/imports/sharding/RDLStatement.g4

```diff
@@ -54,6 +54,10 @@ dropShardingBindingTableRules
 dropShardingBroadcastTableRules
     : DROP SHARDING BROADCAST TABLE RULES
     ;
+    
+dropShardingAlgorithm
+    : DROP SHARDING ALGORITHM algorithmName (COMMA algorithmName)*
```

Comment:
Does we need support `from schema`?

Reply:
I don't think so.   'Sharding Algorithm' is part of sharding rule, and 'Drop Sharing Rule' does not support from schema yet.